### PR TITLE
[SE-3333] Update default branch to juniper.3

### PR DESCRIPTION
**Pull request by toxinu**
This pull request is about updating the default branch to `juniper.3`.

**Jira ticket**

- [SE-3333](https://tasks.opencraft.com/browse/SE-3333)

**Dependencies**

This PR depends on: open-craft/ansible-secrets#209

**Testing instructions**
1. Deploy the changes and verify that a working Juniper (`juniper.3`) instance is spawned.

**Author notes and concerns**

n/a

**Reviewers**
- [ ] @nizarmah

### CODE REVIEW

File: documentation/howtos.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ ECOMMERCE_REPOS:
     REPO: 'ecommerce.git'
     # Use a branch which contains this fix:
     # https://github.com/open-craft/ecommerce/pull/13/files
-    VERSION: opencraft-release/juniper.2
+    VERSION: opencraft-release/juniper.3
```
Comment (nizarmah) [inline documentation/howtos.md:57]: @toxinu I checked the https://github.com/open-craft/ecommerce fork on OpenCraft, and it doesn't seem to have Juniper 3 yet. Would you prefer to create a branch for it, or would you prefer to keep it as Juniper 2?

Comment (toxinu) [reply documentation/howtos.md:57]: @nizarmah Right! I will create the `juniper.3` branch for `ecommerce` and cherry-picked the new commits. 👍🏻

Comment (nizarmah) [reply documentation/howtos.md:57]: Ping me for the review :+1: But please add testing instructions because I am not sure how to exactly test the changes.

**Review by nizarmah — APPROVED** [review]
:+1:

- [x] I tested this:
  - Checked the CircleCI Builds
  - Ran the tests locally
- [x] I read through the code
- [x] I checked for accessibility issues
- [x] Includes documentation
- [ ] ~~I made sure any change in configuration variables is reflected in the corresponding client's `configuration-secure` repository.~~

**Pull request merged**